STYLE-GUIDE.md

```diff
@@ -0,0 +1,22 @@
+# Exercism's Style Guide
+
+This document acts as a Style Guide for the language and wording used in exercises.
+
+## General principle
+
+All content should be written in US english. 
```

Comment:
Can we add a rule that all units must be SI units or SI-compatible, though?

Reply:
Yes: https://en.wikipedia.org/wiki/Non-SI_units_mentioned_in_the_SI + SI please.

Reply:
Yep. Sounds good. (This is actually a potential example where the "exception" thing mentioned below fits in - if an example is about converting from one unit to another and we deliberately choose not to use `si` units).

@SaschaMann Either add a suggestion to this conversation or follow up with a seperate PR (the later is probably preferable so we can get this one over the line quicker)

Reply:
```suggestion
All content should be written in US English.
All units of measurement must be [SI](https://en.wikipedia.org/wiki/International_System_of_Units) or [SI-derived](https://en.wikipedia.org/wiki/SI_derived_unit) units.
```